integration/src/test/scala/com/spotify/scio/bigquery/types/BigQueryStorageIT.scala

```diff
@@ -50,8 +50,9 @@ class BigQueryStorageIT extends AnyFlatSpec with Matchers {
         dt.toLocalDate.plusDays(i),
         dt.toLocalTime.plusMillis(i),
         dt.toLocalDateTime.plusMillis(i),
-        s"POINT($i $i)", // geography is not an avro logical type
-        s"""{"value":$i}""" // json is not an avro logical type
```

Comment:
I found out that BQ adds custom `sqlType` property to the schema. We can reconstruct the original type